### PR TITLE
Migrate app and slides to Gatsbyjs

### PR DESCRIPTION
**Pull request by okonet**
1. Allows publishing slides under `/slides`
1. Uses same start / build setup for both app and slides

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+			resolve: 'gatsby-theme-mdx-deck',
+			options: {
+				// source directory
+				contentPath: './slides',
+				// base path for routes generate by this theme
+				basePath: '/slides',
+			},
+		},
+		{
+			resolve: 'gatsby-plugin-compile-es6-packages',
```
Comment (sapegin) [inline gatsby-config.js:14]: That's where you end up when you try to do zero config 🤦

File: src/gatsby-plugin-theme-ui/index.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export default {
```
Comment (sapegin) [inline src/gatsby-plugin-theme-ui/index.js:1]: Is it used for slides?

**Review by sapegin — APPROVED** [review]

File: gatsby-config.js (hunk 1 above)
Comment (okonet) [reply gatsby-config.js:14]: 🤷🏻‍♂️

File: src/gatsby-plugin-theme-ui/index.js (hunk 2 above)
Comment (okonet) [reply src/gatsby-plugin-theme-ui/index.js:1]: Yes slides need that but it will be applied to all pages

**Pull request merged**